osi_lane.proto

```diff
@@ -155,6 +133,9 @@ message Lane
 //
 message LaneBoundary
 {
+    //The id of the lane boundary.
```

Comment:
Again, if we want to technically remain backward compatible, the id field need to use a new tag number and the remaining fields should retain their tag numbers.